backend/src/index.js

```diff
@@ -1,3 +1,4 @@
+require("dotenv").config()
```

Comment:
I would say, as we may need environment variables from `.env` in all our app, we can just include it in this main file (`index.js`). No need to store this `require` in a `dotenv` variable as you were doing, because we're not using that variable anymore anywhere.